### PR TITLE
Desktop app v5.0.4 docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/desktop-app-changelog.rst [hunk 1]
```diff
@@ -30,6 +30,9 @@ Latest Mattermost Desktop App releases:
 Release v5.0
 --------------
 
+- **v5.0.4, release 2022-02-04**
+ - Fixed an issue where Desktop App toast notifications did't work in v5.0.3.
```
Comment (cwarnermm) [inline source/install/desktop-app-changelog.rst:34]:
```suggestion
 - Fixed an issue where Desktop App toast notifications didn't work in v5.0.3.
```

File: source/install/desktop-app-changelog.rst [hunk 2]
```diff
@@ -30,6 +30,9 @@ Latest Mattermost Desktop App releases:
...
+ - Restored 'Minimize to tray' option for Windows, and added the ability to override the tray icon colour.
```
Comment (cwarnermm) [inline source/install/desktop-app-changelog.rst:35]:
```suggestion
 - Restored **Minimize to tray** option for Windows, and added the ability to override the tray icon color.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**